RestCommClient/Classes/Media/MediaWebRTC.m

```diff
@@ -182,7 +180,7 @@ - (void)connect:(NSString*)sofia_handle sdp:(NSString*)sdp isInitiator:(BOOL)ini
                     [_turnClient requestServersWithCompletionHandler:responseBlock];
                     break;
                
-                case kMXirsys3:
+                case kXirsysV3:
                     turnRequestURL = [NSURL URLWithString:@"https://ice.restcomm.io/_turn/restcomm"];
```

Comment:
@ognjenns we need to pass some parts of the V3 url dynamically, sorry for last minute update, but realized as I was working on the same for Android. This is the format we need (otherwise if somebody else is using V3 outside telestax organization, it won't be able to be configured because the restcomm part is hardcoded):

```
[NSString stringWithFormat:@"%@/%@",
   [_parameters objectForKey:@"turn-url"],  // for example https://ice.restcomm.io/_turn/
   [_parameters objectForKey:@"ice-domain"],  // for example restcomm
];
```

Notice that the second key (i.e. ice-domain) we currently don't expose in the SDK, so we need to introduce.

Do you think you can tackle exposing this new key, ice-domain, in RCDevice API as part of this sprint's side time? If not we can defer it at later time and hardcode it for now as follows:

```
[NSString stringWithFormat:@"%@/%@",
   [_parameters objectForKey:@"turn-url"],  
   @"restcomm",
];
```

Reply:
Ok, will change.

Reply:
done.